src/folding/hypernova/circuit.rs

```diff
@@ -42,15 +43,60 @@ impl<F: PrimeField> SumMulsGammaPowsEqSigmaGadget<F> {
     }
 }
 
+/// Gadget to compute $\Sigma_{i=1}^{q} c_i * \Pi_{j \in S_i} theta_j$.
+pub struct SumCiMulProdThetajGadget<F: PrimeField> {
+    _f: PhantomData<F>,
+}
+
+impl<F: PrimeField> SumCiMulProdThetajGadget<F> {
+    /// Computes $\Sigma_{i=1}^{q} c_i * \Pi_{j \in S_i} theta_j$
```

Comment:
just a small thing that I just noticed (applies to the other part of code done in the previous PRs)
usually
`$\Sigma,~ \Pi$`: ($\Sigma, \Pi$) are used for values,
whereas for the sum and product can use `$\sum, \prod$`: ($\sum,~ \prod$)

But it's just a very small 'stylish' thing that does not affect the code, so if you prefer to use `\Sigma, \Pi` it's fine too.

Reply:
Yes def, thanks for the heads up! Changed at [bdfd7bd](https://github.com/privacy-scaling-explorations/folding-schemes/pull/56/commits/bdfd7bd93ee38e20f06498581f517ad243e4d6f2)